api/HTMLBaseFontElement.json

```diff
@@ -82,7 +82,8 @@
               "version_added": false
             },
             "safari": {
-              "version_added": null
+              "version_added": "6",
+              "version_removed": "10"
```

Comment:
If this and the other properties were removed, was the whole HTMLBaseFontElement interface removed maybe?